### DISCUSSION TITLE
Are hashtag timelines considered public timelines?

### DISCUSSION THREAD

**Discussion by brendanjones** [mastodon/mastodon#26730] [Q&A] [answered]
I assume they are, I'd just like it confirmed because I can't find it in documentation. e.g. it's not mentioned here https://docs.joinmastodon.org/user/posting/#privacy.

**Comment by trwnh** [comment, marked as answer]
Yes. "Public timelines" includes local, federated, and hashtag timelines. (FWIW: The user-facing documentation you linked to could use an update, as it was written for v3.x and probably has a few things that could be added to clarify further.)

**Reply by brendanjones** [reply]
Thanks! The outdated documentation was indeed one reason I was asking, I was trying to make sure I understand everything correctly before contributing to the page. Also for updating the text in-app, which is unclear (which is of course what we were discussing [here](https://github.com/mastodon/mastodon/issues/23038#issuecomment-1697492762) :) ) 

I was thinking the [table](https://docs.joinmastodon.org/user/posting/#privacy) in the documentation could really use an update. If you've got the time and inclination, would you be able to see if I've got everything right in this [table](https://docs.google.com/spreadsheets/d/1Fud7r9P7ShAFAeeOWFdKmwjOhn5csWSJlsAjdypV-0E/edit?usp=sharing) (link to google sheet) I've made?
